src/pages/Landing/LandingContents/Content.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import LandingContents from '../LandingContents/LandingContents';
+
+const Content = ({ firstContentsList, setFirstContentsList }) => {
+  useEffect(() => {
+    fetch('data/contentslist.json')
+      .then(res => res.json())
+      .then(data => {
+        setFirstContentsList(data);
+      });
+  }, [setFirstContentsList]);
+
+  return (
+    <div className="firstContents">
+      <div className="firstTop">
+        <span>기획전</span>
+        <Link to="/">
+          <button className="firstTopBtn" />
+        </Link>
+      </div>
+      <ul className="firstBottom">
+        {firstContentsList.map(contentList => {
```

Comment:
```suggestion
        {firstContentsList.map(content => {
```

contentList 안의 하나의 요소들은 content라고 부르는게 이치에 맞지 않을까요? 리스트 안에 또 리스트가 있다고 착각할만한 이름 같습니다.

이름 항상 고민하시고 의미가 잘 드러나게 지어주세요!